src/parser/test/ParserTest.cpp

```diff
@@ -94,6 +94,33 @@ TEST(Parser, Go) {
     }
 }
 
+TEST(Parser, SpaceOperation) {
+    {
+        GQLParser parser;
+        std::string query = "CREATE SPACE default_space(partition_num=9, replica_factor=3)";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "CREATE SPACE default_space(partition_num=9, replica_factor=3)";
+        auto result = parser.parse(query);
+        ASSERT_FALSE(result.ok());
+    }
+    {
+        GQLParser parser;
+        std::string query = "CREATE SPACE space_without_options()";
```

Comment:
not parentheses

Reply:
pending [Allow to create graph space without options](https://github.com/vesoft-inc/nebula/pull/408)

Reply:
Yeah, you can go to see the grammar about create space in PR #408